src/Components/LandingPage/LandingPage.tsx

```diff
@@ -45,15 +53,62 @@ export const LandingPage = () => {
     }
     setActiveTabKey(tabIndex);
   };
+  const [selectedBlueprint, setSelectedBlueprint] = useState<string>('');
+  const { data: blueprints, isLoading } = useGetBlueprintsQuery({});
 
   const edgeParityFlag = useFlag('edgeParity.image-list');
+  const experimentalFlag =
+    useFlag('image-builder.new-wizard.enabled') || process.env.EXPERIMENTAL;
+
   const traditionalImageList = (
-    <section className="pf-l-page__main-section pf-c-page__main-section">
-      <Quickstarts />
+    <>
+      <PageSection>
+        <Quickstarts />
+      </PageSection>
+      <PageSection>
+        <ImagesTable />
+      </PageSection>
+    </>
+  );
 
-      <ImagesTable />
-    </section>
+  const experimentalImageList = (
+    <>
+      <PageSection>
+        <Quickstarts />
+      </PageSection>
+      <PageSection>
+        <Sidebar
+          hasBorder
+          hasGutter
+          className="pf-v5-u-background-color-100 pf-v5-u-p-lg"
```

Comment:
What does this CSS do? Can we eliminate it?

Reply:
I used the `width` prop instead of `pf-v5-u-p-lg` but for the background color it makes sense to use a class.